hawk/src/main/java/com/orhanobut/hawk/HawkBuilder.java

```diff
@@ -169,9 +170,11 @@ public void build(final Callback callback) {
   public Observable<Boolean> buildRx() {
     HawkUtils.checkRx();
     return Observable.defer(new Func0<Observable<Boolean>>() {
-      @Override public Observable<Boolean> call() {
+      @Override
```

Comment:
one single annotations should be in the same line, I think this is done because of auto-format :)